### PR TITLE
Updated personal information, people.yml

### PR DESCRIPTION
**Pull request by raidun99**
I added information to people.yml.

### CODE REVIEW

**Comment by raidun99** [comment]
@sbillinge Ready to merge

**Comment by sbillinge** [comment]
@raidun99 Nicely done on the PR.  That's a fairly anodyne bio though.....  OK if you want to keep it that way but ok if you want to spruce it up with a bit about yourself too.  I will wait to merge till I hear back....

File: db/people.yml [hunk 1]
```diff
@@ -2721,7 +2721,8 @@ raidun:
   active: true
   aka: []
   avatar: https://github.com/raidun99.png
-  bio: bio
+  bio: Ruby grew up in Albuquerque, NM. She is currently an undergraduate student studying Materials Science and
+  Engineering with a minor in Computer Science. In her spare time she enjoys baking and listening to podcasts.
```
Comment (sbillinge) [inline db/people.yml:2725]: incorrect indentation here....put in two more whitespaces

**Review by sbillinge — COMMENTED** [review]
pls see comment

**Comment by sbillinge** [comment]
nearly there @raidun99

**Pull request merged**